app/views/capts/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<section class="section">
+<div class="container">
+<div class="columns">
+<% if user_signed_in? %>
+<%= render 'profile' %>
+<% else %>
+<%= render 'trends' %>
+<% end %>
```

Comment:
- [ ] put the if statement in a helper method that you call here instead and render the partials within the helper method to remove the if/else logic from the view